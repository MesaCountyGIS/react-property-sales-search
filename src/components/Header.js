import React from 'react';
import logo from '../images/logo.jpg';

const HeaderBackground = {
      backgroundColor: "#013E6B",
      height: "100px",
      color: "white"
}

const Header = () =>
  (
      <div style={HeaderBackground} className="header">
          <a href="http://www.mesacounty.us"><img style={{"padding-top" : "0.15em"}} src={logo} className="App-logo" alt="logo" /></a>
      </div>
  );


export default Header;
