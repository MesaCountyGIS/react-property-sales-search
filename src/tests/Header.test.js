import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe("<Header />", ()=>{
    test('renders without crashing', () => {
        const tree = renderer.create(
            <Header />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
