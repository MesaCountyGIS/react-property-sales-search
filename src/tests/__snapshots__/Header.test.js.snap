// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing 1`] = `
<div
  className="header"
  style={
    Object {
      "backgroundColor": "#013E6B",
      "color": "white",
      "height": "100px",
    }
  }
>
  <a
    href="http://www.mesacounty.us"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.jpg"
      style={
        Object {
          "paddingTop": "0.15em",
        }
      }
    />
  </a>
</div>
`;
